Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -485,11 +485,33 @@ void WebPageProxy::ProcessActivityState::takeCapturingActivity()
     m_isCapturingActivity = m_page.process().throttler().foregroundActivity("View is capturing media"_s).moveToUniquePtr();
 }
 
+#if PLATFORM(MAC)
+// On macOS, we opt out pages out of process suspension if they used the Notification API to avoid breaking
```

Comment:
out pages out

Reply:
You're letting me choose which `out` to keep, risky! :D